### ISSUE TITLE
Invalid argument Help :/ 

### ISSUE THREAD

**Issue by jimmyr57000** [cwaffles/ezpptp#6]
Hello, 

I have this error : http://prnt.sc/axon5j

Could you help me ?

System : Debian 7 x64

**Comment by esemeniuc** [comment]
Are you running this on an OpenVZ system? If so, did you enable PPP?

**Comment by jimmyr57000** [comment]
Yes OpenVZ

Enable PPP ?

**Comment by esemeniuc** [comment]
Yes, please see the image attached if your control panel is SolusVM

![PPP](http://i.imgur.com/DGkg42d.png)
